src/EnergyPlus/SingleDuct.cc

```diff
@@ -1219,7 +1219,7 @@ void GetSysInput(EnergyPlusData &state)
         state.dataSingleDuct->sd_airterminal(state.dataSingleDuct->SysNumGSI).ReheatAirOutletNode =
             state.dataSingleDuct->sd_airterminal(state.dataSingleDuct->SysNumGSI).OutletNodeNum;
         state.dataSingleDuct->sd_airterminal(state.dataSingleDuct->SysNumGSI).MaxAirVolFlowRate = Numbers(1);
-        state.dataSingleDuct->sd_airterminal(state.dataSingleDuct->SysNumGSI).ZoneMinAirFracDes = 0.0;
+        state.dataSingleDuct->sd_airterminal(state.dataSingleDuct->SysNumGSI).ZoneMinAirFracDes = 1.0;
```

Comment:
For constant volume terminal units this value is used to set the air flow used for sizing the heating coil. If this value is 0 then the air flow rate used to size the heating coil is always equal to NonAirSysDesHeatVolFlow when is the zone design air flow rate, which could be significantly different from the flow rate of the TU(e.g. user specified flow, etc.). This value was changed to 1 for all TUs that did not have an input for Minimum Air Flow Fraction. The NoReheat version does not have a heating coil and this wouldn't matter, but for consistency, all were changed.

                    TermUnitSizing(CurTermUnitSizingNum).AirVolFlow =
                        max(TermUnitFinalZoneSizing(CurTermUnitSizingNum).NonAirSysDesHeatVolFlow,
                            this->MaxAirVolFlowRate * this->ZoneMinAirFracDes * this->ZoneTurndownMinAirFrac);

Reply:
I agree.

Reply:
@rraustad Ok, I see how this makes sense. For singleduct, an alternative fix would be to add an if block for the 2 constant volume terminal units [here](https://github.com/NREL/EnergyPlus/blob/b03bdf040164a5ca370b6ea69bf41e08423ad0f6/src/EnergyPlus/SingleDuct.cc#L3493-L3504) similar to the one a few lines below. Or maybe not, maybe that if can be removed from lines 3502-03 because `this->ZoneMinAirFracDes * this->ZoneTurndownMinAirFrac` now = 1 for these terminal unit types?

In general, this sizing function has lots of overlapping and/or incomplete paths, sometimes testing on `SysType_Num`, sometimes `DamperHeatingAction` (but this doesn't always (ever?) have a case for `HeatingActionNotUsed` which these terminal unit types are set to, sometimes on `ZoneMinAirFracMethod`, sometimes these are nested. I'm not suggesting this be refactored here, just saying.  

Now to look at the big diffs more closely to be certain they are as desired.